src/amo/components/Search/index.js

```diff
@@ -120,6 +124,18 @@ export class SearchBase extends React.Component {
       <SearchSort filters={filters} pathname={pathname} />
     ) : null;
 
+    if (!filters.query || filters.query.length === 0) {
```

Comment:
Are we planning category wise search?
Because for now we get addons from categories by https://addons.mozilla.org/en-US/firefox/extensions/photos-music-videos/